### PR TITLE
babble-bubbles:0.1.0

### PR DESCRIPTION
**Pull request by ShadowMitia**
I am submitting
- [X] a new package
- [ ] an update for a package

I have read and followed the submission guidelines and, in particular, I
- [X] selected a name in conformance with the guidelines
- [X] added a `typst.toml` file with all required keys
- [X] added a `README.md` with documentation for my package
- [X] have chosen a license and added a `LICENSE` file or linked one in my `README.md`
- [X] ensured that my submission does not infringe upon the rights of a third party
- [X] tested my package locally on my system and it worked
- [X] named this PR as `name:version` of the submitted package
- [X] agree that my package will not be removed without good reason

Description: 

This is a convenience package to create callouts in typst, inspired by the [Obsidan](https://obsidian.md/) callouts.

### CODE REVIEW

**Comment by laurmaedje** [comment]
The name is a bit too descriptive as-is, could add a little bit of spice to it, so that other potential packages for callouts have a level playing field?

I would also suggest using kebab-case, at least for the public function signatures, but ultimately it's up to you. :)

**Comment by ShadowMitia** [comment]
> The name is a bit too descriptive as-is, could add a little bit of spice to it, so that other potential packages for callouts have a level playing field?

Oh I didn't think about that... Sorry... Sure I'll find something!

> 
> I would also suggest using kebab-case, at least for the public function signatures, but ultimately it's up to you. :)

I'm not sure I understand that one, do you mean having something like `callout-warning`?

**Comment by laurmaedje** [comment]
I meant that wherever you use snake_case (e.g. `title_color`), kebab-case (i.e. `title-color`) might be more appropriate because its the idiomatic Typst casing convention. However, it's not a hard and fast rule and I'm aware than many people use snake_case out of habit, so I don't force anybody to change it, just a suggestion!

**Comment by ShadowMitia** [comment]
Oh I see!
I guess I have rust memory muscle in this case^^
I don't mind changing if you think its more idiomatic.

**Comment by ShadowMitia** [comment]
Changed the name, and moved to kebab-case.
Let me know if there's anything else!

File: packages/preview/babble-bubbles/0.1.0/README.md [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+
+Create a default callout.
+Tweak the parameters to create your own!
+
+```typst
+callout(
+  body,
+  title: "Callout",
+  fill: blue,
+  title_color: white,
```
Comment (laurmaedje) [inline packages/preview/babble-bubbles/0.1.0/README.md:58]: looks like the example is not yet updated

File: packages/preview/babble-bubbles/0.1.0/README.md [hunk 2]
```diff
@@ -0,0 +1,83 @@
...
+  icon: none)
+```
+
+### Tips
+
+You can create aliases to more easily handle your
+newly create callouts
+
+```
+#let mycallout(body) = callout(title: "My callout", body)
```
Comment (laurmaedje) [inline packages/preview/babble-bubbles/0.1.0/README.md:69]:
```suggestion
#let mycallout = callout.with(title: "My callout")
```

File: packages/preview/babble-bubbles/0.1.0/README.md [hunk 3]
```diff
@@ -0,0 +1,83 @@
...
+
+#mycallout[Hey this is my custom callout!]
+```
+
+If you want to still to pass parameters use this shorthand:
+
+```
+// Note the added `params` arguments!
+#let mycallout(body, ..params) = callout(title: "My callout", body, ..params)
```
Comment (laurmaedje) [inline packages/preview/babble-bubbles/0.1.0/README.md:78]:
```suggestion
#let mycallout = callout.with(title: "My callout")
```
Actually, you can now completely remove this section I think. `.with` just does the job.

**Comment by ShadowMitia** [comment]
All done!
I wasn't sure about the `with` method, but that simplifies the package and the user api, thanks! :smiley:

**Comment by laurmaedje** [comment]
Great, thanks!

**Pull request merged**